test/CMakeLists.txt

```diff
@@ -19,6 +19,9 @@
 cmake_minimum_required(VERSION 3.19)
 project(size_test)
 
+# Use C++11 for size test.
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
why the inconsistency between this and the next file (i.e. build_size_test.sh). namely, it is setting 11 and the other one is setting 14.

Reply:
I guess, there are still some changes we have to make for libexecutorch to compile with c++11. Size test binary can compile using c++11, so we can use it.